### PR TITLE
httpie: 1.0.3

### PR DESCRIPTION
**Pull request by jeffbyrnes**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Takes over for #43471

### CODE REVIEW

**Review by chenrui333 — APPROVED** [review]
Thanks for taking care of this!

File: Formula/httpie.rb [hunk 1]
```diff
@@ -3,38 +3,38 @@ class Httpie < Formula
...
-  url "https://files.pythonhosted.org/packages/09/8d/581ef7bd9a09dc30b621638a4fa805a2073bbfb45fa06ed37f998f172419/httpie-1.0.2.tar.gz"
-  sha256 "fc676c85febdf3d80abc1ef6fa71ec3764d8b838806a7ae4e55e5e5aa014a2ab"
+  url "https://files.pythonhosted.org/packages/d5/a4/ab61c1dbfdef33c7b7f5f7df0d79eb5cd55a106601a4acc17f983f320b4a/httpie-1.0.3.tar.gz"
+  sha256 "6d1b6e21da7d3ec030ae95536d4032c1129bdaf9de4adc72c596b87e5f646e80"
   head "https://github.com/jakubroztocil/httpie.git"
 
   bottle do
     cellar :any_skip_relocation
     rebuild 1
-    sha256 "4b93be77eb04fbdd3af064849b182520383676b7b03728d94f6867f250d2e35e" => :mojave
```
Comment (fxcoudert) [inline Formula/httpie.rb]: Do not modify the bottle block

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/httpie.rb (hunk 1 above)
Comment (jeffbyrnes) [reply Formula/httpie.rb]: So, I saw that in the contribution guidelines, though it’s countered by the fact that the script in the httpie project does modify these things (and remove the `rebuild 1` portion, too).

You can see the results in this commit I made over on that project: jakubroztocil/httpie#801

Should I simply ignore what the `make brew-deps` command on that project generated?

Comment (SMillerDev) [reply Formula/httpie.rb]: If it modified the bottle block, yes

Comment (fxcoudert) [reply Formula/httpie.rb]: > the script in the httpie project does modify these things

Modifying the resources blocks is right. Modifying the `bottle do` block is wrong.

Comment (jeffbyrnes) [reply Formula/httpie.rb]: Would you expand on that, @fxcoudert? I’m not entirely familiar with how bottles are generated, frankly.

Comment (jeffbyrnes) [reply Formula/httpie.rb]: Would you expand on that, @fxcoudert? I’m not entirely familiar with how bottles are generated, frankly.

Comment (fxcoudert) [reply Formula/httpie.rb]: The bottles are generated by our CI, and the bottle block is modified automatically when we merge a PR. Pull requests should always leave the bottle block untouched.

Comment (jeffbyrnes) [reply Formula/httpie.rb]: This makes way more sense! I’ll fix this up, stand by…

**Comment by fxcoudert** [comment]
Thanks @jeffbyrnes, I am merging now.
Please note our preferred commit message for updates is `httpie 1.0.3` (without the `:`)

**Pull request closed without merging**